src/sysusers/sysusers.c

```diff
@@ -292,6 +292,7 @@ static int putgrent_with_members(const struct group *gr, FILE *group) {
         return 0;
 }
 
+#ifdef HAVE_GSHADOW
```

Comment:
Meson is not patched to set this, so this patch would disable shadow group for those builds.